packages/validation/src/Review/index.ts

```diff
@@ -1,7 +1,7 @@
 import * as Yup from 'yup';
 
 export const proposalGradeValidationSchema = Yup.object().shape({
-  comment: Yup.string().max(500, 'Too long comment').nullable(),
+  comment: Yup.string().max(1000, 'Too long comment').nullable(),
```

Comment:
I hope this is enough characters now and you discussed this with @bolmsten about increasing to 1000 characters.